system-test/spanner.ts

```diff
@@ -65,29 +65,45 @@ describe('Spanner', () => {
   };
   const IS_EMULATOR_ENABLED =
     typeof process.env.SPANNER_EMULATOR_HOST !== 'undefined';
+  const RESOURCES_TO_CLEAN: Array<Instance | Backup | Database> = [];
+  const DATABASE = instance.database(generateName('database'));
+  RESOURCES_TO_CLEAN.push(DATABASE);
 
   before(async () => {
     if (generateInstanceForTest) {
       await deleteOldTestInstances();
       const [, operation] = await instance.create(INSTANCE_CONFIG);
       await operation.promise();
+      RESOURCES_TO_CLEAN.push(instance);
     } else {
+      // Update 'created' label timestamp on an existing instance
+      const [meta] = await instance.getMetadata();
+      const labels = Object.assign({}, meta.labels, {created: CURRENT_TIME});
+      await instance.setMetadata({labels});
       console.log(
         `Not creating temp instance, using + ${instance.formattedName_}...`
       );
     }
+    const [, operation] = await DATABASE.create({
+      schema: `
+          CREATE TABLE Singers (
+            SingerId STRING(1024) NOT NULL,
+            Name STRING(1024),
+          ) PRIMARY KEY(SingerId)`,
+    });
+    await operation.promise();
   });
 
   after(async () => {
     if (generateInstanceForTest) {
       await deleteTestInstances();
+    } else {
```

Comment:
That makes sense! Would you mind leaving a comment in the code to explain why we exclude those?

Also, it might be cleaner to get it down to one filter & map:

```js
await Promise.all(
  RESOURCES_TO_CLEAN.filter(
    // No need to delete databases as we will delete the instance
    resource => !(resource instanceof Database)
  ).map(resource => resource.delete())
);
```

Reply:
Good point. Adding in a "sort()" would probably be overkill, so no worries. Change looks good, thank you!